link-grammar/prune.c

```diff
@@ -573,14 +573,14 @@ static inline bool matches_S(connector_table *ct, Connector * c, char dir)
  */
 static int mark_dead_connectors(connector_table *ct, Exp * e, char dir)
 {
-	int count;
-	count = 0;
+	int count = 0;
+	Connector dummy;
+	init_connector(&dummy);
```

Comment:
I'm not surprised that LTO makes no difference -- everything that's been important has already been optimized by hand.   The smaller binary is surprising.   I'm guessing that maybe some of the debuggiung symbols are no longer being generated, or somehow got simplified.

LTO can maybe ever-so-slightly reduce binary sizes, by removing some of the subroutine glue linkage.  Maybe some symbol table sizes can decrease in size...